.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+# Description
+
+Thank you for your contribution!
+
+Please provide a summary of the changes and the motivation behind the change.
```

Comment:
I think you want to put this into a comment, so that it's not automatically pulled into the PR description without being deleted.
See: https://raw.githubusercontent.com/openssl/openssl/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Done!

Reply:
Done!